chart/templates/deployment.yaml

```diff
@@ -81,6 +81,11 @@ spec:
         - containerPort: 444
           hostPort: {{ int .Values.hostPort }}
           protocol: TCP
+{{- end}}
+{{- if .Values.imperativeExtensionApi.enabled }}
+        - containerPort: 6666
+          hostPort: 6666
+          protocol: TCP
```

Comment:
I'm fairly confident that this can be removed once Felipe's work on RemoteDialerProxy is completed. Perhaps we want to add a comment to remind us that we need to remove this before the release.